src/styles/card-block.scss

```diff
@@ -0,0 +1,106 @@
+
+.card {
+  max-width: $card-width;
+
+  background: $white-color;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__heading {
+    margin-bottom: 4px;
+
+    display: flex;
+
+    font-size: 12px;
+    font-weight: $medium-text;
+    line-height: 18px;
+    color: $dark-blue;
+  }
+  &__text-small {
+    color: $grey-color;
+
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $grey-color;
+
+    display: flex;
+    align-items: flex-end;
+  }
+
+  &__price-cost {
+    font-weight: $bold;
+    font-size: 16px;
+    color: $dark-blue;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+
+    border-radius: 5px;
+    transition: color $duration,
+      border-color $duration,
+      background-color $duration;
+    color: $white-color;
+    background: $blue-color;
+    border: 1px solid transparent;
+
+    text-decoration: none;
+    font-weight: $bold;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+
+    &:hover {
+      border-color: $blue-color;
+      color: $blue-color;
+      background-color: $white-color;
+    }
+  }
+
+}
+
+.catalog__card {
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+
+  object-fit: cover;
+  object-position: center;
+  margin-bottom: 40px;
+}
```

Comment:
Should it be a BEM block? I don't think we will want to reuse image with this exact styles somewhere else
Also can we use margins for BEM-blocks?